webservices/templates/swagger-ui.html

```diff
@@ -30,16 +29,102 @@
 </head>
 
 <body>
-    <div id="swagger-ui"></div>
-    <script src="{{ swagger_static('dist/swagger-layout.js') }}"> </script>
+<div id="swagger-ui"></div>
+
+ <script type="text/javascript">
+   /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+   var apiUmbrellaSignupOptions = {
+     // Pick a short, unique name to identify your site, like 'gsa-auctions'
+     // in this example.
+     registrationSource: 'fec-api',
+
+     // Enter the API key you signed up for and specially configured for this
+     // API key signup embed form.
+     apiKey: "{{ api_key_signup_key }}",
+     // document.write(apiKey.toLocaleString( ));
+     // Provide a URL or e-mail address to be used for customer support.
+     //
+     // The format for e-mail addresses can be given as either
+     // 'example@example.com' or 'mailto:example@example.com'.
+     contactUrl: 'APIinfo@fec.gov',
```

Comment:
The email sent out to the user after successfully signing up for a new API key has contactus email still pointing to `api.data.gov`. It should be apiinfo@fec.gov. if you change the contractUrl value to `mailto:APIinfo@fec.gov`, then the response email shows  fec email email address.

Reply:
Thank you @pkfec!! This is GREAT! I will fix it.